docs/bokeh/source/docs/user_guide/topics/categorical.rst

```diff
@@ -71,73 +92,76 @@ This will shift each bar horizontally by the corresponding offset.
-of categories will produce a *categorical heatmap*. Such a plot has two
+of categories will produce a *categorical heatmap*. This is a plot with two
 categorical axes.
 
 The following plot lists years from 1948 to 2016 on its x-axis and months of
 the year on the y-axis. Each rectangle of the plot corresponds to a
 ``(year, month)`` pair. The color of the rectangle indicates the rate of
 unemployment in a given month of a given year.
 
-This example uses the ``LinearColorMapper`` to map the colors of the plot
-because the unemployment rate is a continuous variable. This mapper is also
-passed to the color bar to provide a visual legend on the right:
+This example uses the :class:`~bokeh.models.mappers.LinearColorMapper` to map the
+colors of the plot because the unemployment rate is a continuous variable.
+This mapper is also passed to the color bar to provide a visual legend on the right:
 
 .. bokeh-plot:: __REPO__/examples/topics/categorical/heatmap_unemployment.py
     :source-position: below
 
-The following periodic table is a good example of the techniques
-in this chapter:
-
-* Color mappers
-* Visual offsets
-* pandas DataFrames
-* Tooltips
+The following periodic table uses several of the techniques in this chapter:
 
 .. bokeh-plot:: __REPO__/examples/topics/categorical/periodic.py
     :source-position: below
 
-When we have more than three to four quantitative variables, it is sometimes more useful
-to quantify the amount of association between pairs of variables and visualize this quantity
-rather than the raw data. One common way to do this is to calculate correlation coefficients.
+Correlograms
+~~~~~~~~~~~~
+
+When you have more than three to four quantitative variables per category, it can be
+more useful to quantify the amount of association between pairs of variables and
+visualize this quantity rather than the raw data.
+One common way to do this is to calculate correlation coefficients.
 Visualizations of correlation coefficients are called correlograms.
 
-The following correlogam is another good example of the techniques in this chapter.
+The following correlogram is another good example of the techniques in this chapter.
 
-In this plot, we display the correlations as colored circles and scale the circle size with
-the absolute value of the correlation coefficient. In this way, low correlations are suppressed
-and high correlations stand out better.
+This plot displays the correlations as colored circles. The scale of the circles
+corresponds to the absolute value of the correlation coefficient.
+This way, low correlations are suppressed and high correlations stand out better.
 
-This example uses ``linear_cmap`` to map the colors of the plot
-in order to highlight the correlations between the pair of elements.
+This example uses the :class:`~bokeh.models.mappers.LinearColorMapper` to map the
+colors of the plot in order to highlight the correlations between the pair of elements.
```

Comment:
I guess this is slightly out of sync, the example was updated to use `linear_cmap` and `construct_colorbar` rathern than `LinearColorMapper` directly.

Reply:
Good point - I'll also update this for the unemployment example description!